app/models/effective/log.rb

```diff
@@ -26,11 +26,15 @@ class Log < ActiveRecord::Base
 
       status              :string, :validates => [:presence, :inclusion => {:in => EffectiveLogging.statuses }]
 
+      read_at             :datetime
       timestamps
     end
 
     default_scope -> { order("#{EffectiveLogging.logs_table_name.to_s}.updated_at DESC") }
 
+    scope :read,   -> { where('read_at IS NOT NULL') }
+    scope :unread, -> { where(read_at: nil) }
```

Comment:
These look pretty backward compatible to me but not sure if they're up to snuff.